build.cake

```diff
@@ -1074,6 +1047,45 @@ Task("Default")
 
 RunTarget(target);
 
+
+
+void SetNunitEnvironmentVariables()
```

Comment:
Do we need this?

Reply:
No... I thought I deleted that :-/ 

pushed a fix